noise/noise.py

```diff
@@ -155,7 +155,7 @@ def sendmsg(node_id, msg, plugin, request, pay=None, **kwargs):
 @plugin.async_method('recvmsg')
 def recvmsg(plugin, request, last_id=None, **kwargs):
     next_id = int(last_id) + 1 if last_id is not None else len(plugin.messages)
-    if next_id < len(plugin.messages):
+    if next_id <= len(plugin.messages):
```

Comment:
This implicitly relies on the fact that the message IDs are strictly incremented by one, which is ok in this case, but may not be in the future. This breaks the API that you specify the last ID that you processed, and we return the next one.